lib/l10n/app_fr.arb

```diff
@@ -164,69 +154,46 @@
       }
     }
   },
-  "offlineMode": "Mode hors ligne",
-  "@offlineMode": {},
-  "internalExternalIpExplanation": "Si vous voulez pouvoir accéder à votre serveur Jellyfin hors de votre réseau local, vous devez utiliser votre IP externe.\n\nSi votre serveur est sur un port HTTP (80/443), vous n'avez pas besoin de spécifier de port. Ce sera surement le cas si votre serveur est derrière un reverse proxy.",
-  "@internalExternalIpExplanation": {
-    "description": "Extra info for which IP to use for remote access, and info on whether or not the user needs to specify a port."
-  },
-  "emptyServerUrl": "L'URL du serveur ne peut pas être vide",
-  "@emptyServerUrl": {
-    "description": "Error message that shows when the user submits a login without a server URL"
-  },
-  "startMixNoSongsAlbum": "Appuyez longuement sur un album pour l'ajouter ou l'enlever du générateur de mix, avant de le lancer",
-  "@startMixNoSongsAlbum": {
-    "description": "Snackbar message that shows when the user presses the instant mix button with no albums selected"
-  },
-  "shuffleAll": "Lecture aléatoire",
-  "@shuffleAll": {},
-  "finamp": "Finamp",
-  "@finamp": {},
-  "artist": "Artiste",
-  "@artist": {},
-  "startMixNoSongsArtist": "Appuyez longuement sur un artiste pour l'ajouter ou l'enlever du générateur de mix, avant de le lancer",
-  "@startMixNoSongsArtist": {
-    "description": "Snackbar message that shows when the user presses the instant mix button with no artists selected"
-  },
-  "productionYear": "Année de production",
-  "@productionYear": {},
-  "premiereDate": "Date de sortie",
-  "@premiereDate": {},
-  "downloadedMissingImages": "{count,plural, =0{Aucune image manquante trouvée} =1{{count} image manquante téléchargée} other{{count} images manquantes téléchargées}}",
-  "@downloadedMissingImages": {
-    "description": "Message that shows when the user downloads missing images",
+  "downloadedItemsImagesCount": "{downloadedItems}, {downloadedImages}",
+  "@downloadedItemsImagesCount": {
+    "description": "Sert à fusionner downloadedItemsCount et downloadedImagesCount car les outils intl de Flutter ne supportent pas plusieurs pluriels dans une seule chaîne de caractères. https://github.com/flutter/flutter/issues/86906",
     "placeholders": {
-      "count": {
-        "type": "int"
+      "downloadedItems": {
+        "type": "String",
+        "example": "12 téléchargements"
+      },
+      "downloadedImages": {
+        "type": "String",
+        "example": "1 image"
       }
     }
   },
-  "dlRunning": "{count} en cours",
-  "@dlRunning": {
+  "dlComplete": "{count,plural,=1{{count} complété} other{{count} complétés}}",
+  "@dlComplete": {
```

Comment:
I guess it's a bad practice to mismatch the order of dlRunning dlComplete etc in French from other translations

Reply:
I've just copied the app_en and translated it, as I saw there was missing field in the french translation uh